universal-editor-task-management/src/universal-editor-task-management/web-src/src/components/ExtensionRegistration.js

```diff
@@ -36,7 +36,13 @@ function ExtensionRegistration() {
           },
         },
       };
-      await register(registrationConfig);
+      const guestConnection = await register(registrationConfig);
+
+      const { WORKFRONT_INSTANCE_URL, IMS_ENV } = guestConnection.configuration;
+      let mfeURLwithConfigParams = `&workfrontInstanceUrl=${WORKFRONT_INSTANCE_URL}`;
```

Comment:
Does it make sense to validate the `WORKFRONT_INSTANCE_URL` param?

Reply:
don't think so, if the url is not valid or not provided, the MFE would throw an error, but don't think the Extension should validate it, I can change it if you think otherwise.

Reply:
Ok, this makes sense if the MFE validates the URL. All good, thanks.